doc/cephadm/troubleshooting.rst

```diff
@@ -110,34 +110,45 @@ ssh errors
 
 Error message::
 
-  xxxxxx.gateway_bootstrap.HostNotFound: -F /tmp/cephadm-conf-kbqvkrkw root@10.10.1.2
-  raise OrchestratorError('Failed to connect to %s (%s).  Check that the host is reachable and accepts  connections using the cephadm SSH key' % (host, addr)) from
-  orchestrator._interface.OrchestratorError: Failed to connect to 10.10.1.2 (10.10.1.2).  Check that the host is reachable and accepts connections using the cephadm SSH key
+  execnet.gateway_bootstrap.HostNotFound: -F /tmp/cephadm-conf-73z09u6g -i /tmp/cephadm-identity-ky7ahp_5 root@10.10.1.2
+  ...
+  raise OrchestratorError(msg) from e
+  orchestrator._interface.OrchestratorError: Failed to connect to 10.10.1.2 (10.10.1.2).
+  Please make sure that the host is reachable and accepts connections using the cephadm SSH key
+
+  To add the cephadm SSH key to the host:
+  > ceph cephadm get-pub-key > ~/cephadm_ssh_key.pub
+  > ssh-copy-id -f -i ~/cephadm_ssh_key.pub root@10.10.1.2
+
+  To check that the host is reachable:
+  > ceph cephadm get-ssh-config > ssh_config
+  > ceph config-key get mgr/cephadm/ssh_identity_key > ~/cephadm_private_key
+  > ssh -F ssh_config -i ~/cephadm_private_key root@10.10.1.2
```

Comment:
https://docs.ceph.com/ceph-prs/36330/cephadm/troubleshooting/#ssh-errors

this looks a bit odd and unreadable. Are you sure this is what users get?